common/djangoapps/config_models/models.py

```diff
@@ -41,7 +41,10 @@ def current_set(self):
         """
         assert self.model.KEY_FIELDS != (), "Just use model.current() if there are no KEY_FIELDS"
         return self.get_queryset().extra(           # pylint: disable=no-member
-            where=["id IN ({subquery})".format(subquery=self._current_ids_subquery())],
+            where=["{table_name}.id IN ({subquery})".format(
```

Comment:
@bdero this `{table_name}.id IN ({subquery}).format()` occurs three times in this file - might make sense to extract it into a helper method (if possible)